app/controllers/locks_controller.rb

```diff
@@ -11,14 +11,18 @@ def index
   end
 
   def create
-    lock = Lock.create!(
-      params.require(:lock).
-        permit(:description, :resource_id, :resource_type, :warning, :delete_in).
-        merge(user: current_user)
-    )
-    respond_to do |format|
-      format.html { redirect_back notice: 'Locked', fallback_location: root_path }
-      format.json { render json: {lock: lock} }
+    new_lock = Lock.new(lock_params)
```

Comment:
also new_lock -> lock ?

Reply:
There's a method `lock` in the controller, and I didn't want to cause confusion by making the variable shadow the method, and `new_lock` is somewhat more descriptive?